servicetalk-test-resources/src/main/java/io/servicetalk/test/resources/DefaultTestCerts.java

```diff
@@ -44,6 +44,15 @@ public static InputStream loadServerPem() {
         return DefaultTestCerts.class.getResourceAsStream("localhost_server.pem");
     }
 
+    /**
+     * Load the server certificate PKCS#12 keystore file.
+     *
+     * @return an {@link InputStream} from the server key/certificate keystore file.
+     */
+    public static InputStream loadServerP12() {
```

Comment:
Will be great to add an example for PKCS#12 later as it might be a bit trickier for users.

Reply:
I agree. I ended up copying the very weak test pattern used for the other resources but believe that more complete tests should be created for each resource.

Reply:
I will create an RFE issue. This would be a good "first contribution" issue.

Reply:
https://github.com/apple/servicetalk/issues/2290

Reply:
I was talking about examples, not tests. Creating an `SslConfig` with pre-existing pem certs is straightforward, users just need input stream. We already do that in examples: https://github.com/apple/servicetalk/blob/14e9fa7e16bbf2512dfb194bc075c917f5293e8c/servicetalk-examples/http/mutual-tls/src/main/java/io/servicetalk/examples/http/mutualtls/HttpClientMutualTLS.java#L37

This approach won't work for PKCS#12. Users have to load them manually and go through the trust/key factory. It's not obvious how to do that.